src/views/GovernanceReviewTeam/Actions/index.test.tsx

```diff
@@ -3,18 +3,47 @@ import { MemoryRouter, Route } from 'react-router-dom';
 import { render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
-import { systemIntake } from 'data/mock/systemIntake';
+import {
+  getSystemIntakeContactsQuery,
+  getSystemIntakeQuery,
+  requester,
+  systemIntake
+} from 'data/mock/systemIntake';
+import { MockedQuery } from 'types/util';
+import VerboseMockedProvider from 'utils/testing/VerboseMockedProvider';
 
-import ChooseAction from '.';
+import Actions from '.';
+
+const renderActionPage = ({
+  mocks,
+  action
+}: {
+  mocks: MockedQuery[];
+  action?: string;
+}) => {
+  return render(
+    <VerboseMockedProvider mocks={mocks}>
+      <MemoryRouter
+        initialEntries={[
+          `/governance-review-team/${systemIntake.id}/actions/${action}`
+        ]}
+      >
+        <Route path={[`/governance-review-team/:systemId/actions/:subPage?`]}>
+          <Actions systemIntake={systemIntake} />
+        </Route>
+      </MemoryRouter>
+    </VerboseMockedProvider>
+  );
+};
 
 describe('IT Gov Actions', () => {
   it('Renders options and selects action', async () => {
-    const { getByRole, getByText, getByTitle, findByRole } = render(
+    const { getByRole, getByText, getByTitle } = render(
```

Comment:
I've started using `screen.{query method}` instead of getting the methods from `render()`. It's convenient for the whole file.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen